totalRP3/tools/Locale.lua

```diff
@@ -592,6 +592,8 @@ Possible status:
 	CO_CHAT_MAIN_OOC_COLOR = "OOC color",
 	CO_CHAT_MAIN_EMOTE_YELL = "No yelled emote",
 	CO_CHAT_MAIN_EMOTE_YELL_TT = "Do not show *emote* or <emote> in yelling.",
+	CO_CHAT_MAIN_SPEECH = "Speech detection",
+	CO_CHAT_MAIN_SPEECH_USE = "Use speech detection",
```

Comment:
Can we add a tooltip text that says what "speech detection" is? Namely, that it'll colour quoted text in white.

Otherwise we'll get some *valued user* who thinks we've implemented some sort of TTS feature or something daft.